### DISCUSSION TITLE
Unable to specify target repository for issue created in Project

### DISCUSSION THREAD

**Discussion by kcgrimes** [community/community#111387] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Projects

### Body

Hi, I have a couple repositories and just started playing with Projects. My "hello world" project was being practiced in while linking to my "hello world" repo, both happen to be my first repo and project.

I then went to make a "real" Project, and notice that when I click to create an issue from within the Project, it says at the top of the dialog "Create new issue in username/hello-world", and does not give me the option to select which repo, as suggested in the docs below:

https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-project

I would like to be able to have at least a drop down as suggested by that link but preferably a default repo that issues from a Project are made in. Project 1 for Repo 1, Project 2 for Repo 2, etc.

I can create a draft then convert it to be an issue to my desired repo. I can create an issue within the repo and assign it to the project.

Free account, personal repo and projects. I've tried copying project, creating from scratch, linking/not linking, logging in and out... can't seem to find anything on this.

Thank you for the help!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by alfonsogarza** [comment]
This was opened at the beginning of the year and sadly got no replies.
https://github.com/orgs/community/discussions/101955

For us its such a big pain and given the silence in that ticket we are considering using Linear.app or other alternatives

**Reply by kcgrimes** [reply]
Thanks for noticing, at this point I totally agree, this is a bug, not a question. Your workaround of filtering by repo worked for me, and for my intentions that is actually a fix. But the ability to specifically select a default +/- a dropdown for repo selection, as is described in the docs, is warranted. Thanks again.

**Comment by azenMatt** [comment, marked as answer]
Hi, thanks for sharing this one - it has now been fixed 😄 

Let me know if it causes you any further problems!

**Reply by kcgrimes** [reply]
Confirmed it is working as intended as of a couple days ago, thank you so much!

**Comment by ssardina** [comment]
Hi @azenMatt . I am sorry to say that this issue seems to be back...

I am experiencing exactly the same problem as @kcgrimes reported, (and the same workaround, cumbersome but works). :cry:

**Reply by azenMatt** [reply]
Thanks for flagging - we will take a look into this...

**Reply by nasquasha** [reply]
hi @ssardina, are you able to type your repository name into the input in the dialog and have it come up as an option?

**Reply by ssardina** [reply]
Hi @nasquasha . Yes when I type the start of the name it comes up. Only one is listed initially but when I type the repo it shows up :ok_hand:

**Reply by nasquasha** [reply]
ok great, in that case this may not be a bug per se but rather unexpected (incomplete) results for initial suggestions. I believe most suggestions take activity into consideration so a new repo that doesn't have issues present in the project or hasn't often been used may not show up for that reason.

**Reply by ssardina** [reply]
Yes, that makes total sense. That's why only repos that have been used in the project already show up. For others you need to spell them out partially. It makes sense. Thanks! :pray:

**Reply by fnowak77** [reply]
I have the same problem! Event spelling the project entirely do not make it appear and/or be "selectable" when creating an issue from Projects.

**Reply by nasquasha** [reply]
@fnowak77 can you tell me more about your project and the repo you're trying to select? is your project public/private? is the repo public/private? are they under the same owner or different? also could you potentially share a video of the behavior you're seeing?

**Reply by emileswain** [reply]
Just got caught out by this as well. Short of changing the ui you might just change the "Select repository" to "Search repository" as this would better infer that you can find all repositories rather than just select what is currently shown, which is just one repo.

**Comment by jamesturner246** [comment]
Still having the same issue as of 6th Aug 2025.

**Reply by ForJobOk** [reply]
I faced the same problem, I can't select the repository.

![GithubProblem](https://github.com/user-attachments/assets/0251d61b-8181-4b4c-8b20-d19ed9b1e7d4)

**Reply by azenMatt** [reply]
@ForJobOk - does the 'back arrow' next to the 'create new issue in....' text not take you back to an option to pick a different repo?

**Reply by ForJobOk** [reply]
I can select a repository to enter # in input field. Thank you!
https://github.com/user-attachments/assets/d6b84a82-e077-4eef-8651-069e04bbc015

**Reply by bostonaholic** [reply]
Thanks, @azenMatt and @ForJobOk. Using the <- or `#` works for selecting the repo I want the issue created in.

But this is still not ideal because I don't want to have to do this for every single issue I create in a project. It should just be the default behavior to create an issue in the repository that the project is linked to.

**Reply by ForJobOk** [reply]
I completely agree with you. Since we've already linked the project, choosing the repository should be optional. Why do you need to explicitly link an issue to a repository every time?
